domain/src/main/java/com.pyshankov.microservices.domain/Product.java

```diff
@@ -1,13 +1,23 @@
 package com.pyshankov.microservices.domain;
 
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
 import java.math.BigDecimal;
 
 /**
  * Created by pyshankov on 4/29/18.
  */
-public class Product {
-    private String id;
+@NoArgsConstructor
+@Data public class Product {
+    private Long id;
```

Comment:
why Long? it's better to use String as id for mongo table 
add @id annotation on id field to indicate mongodb generate id